fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -1584,32 +1633,26 @@ Future<std::vector<std::pair<TenantName, TenantMapEntry>>> listTenants(
 	loop {
 		try {
 			tr->setOption(FDBTransactionOptions::READ_SYSTEM_KEYS);
-			tr->setOption(FDBTransactionOptions::READ_LOCK_AWARE);
```

Comment:
Did you mean to remove `READ_LOCK_AWARE`?

Reply:
Yeah, the same reason for the one in `runTransaction()`

Reply:
In this case we aren't calling `commit`, so does it still run into trouble? If so, maybe we could use `LOCK_AWARE` instead.

Reply:
NVM, my fault. I thought listTenantsTransaction called `runTransaction`. I confused it a and `TenantAPI::listTenantMetadata`